packages/backup/actions.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Action Hooks for Jetpack Backup module.
+ *
+ * @package automattic/jetpack-backup
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	return;
+}
+
+/**
+ * Cleanup Helper Scripts from a scheduled event after installing them.
+ */
+function jetpack_backup_cleanup_helper_scripts() {
+	Automattic\Jetpack\Backup\Helper_Script_Manager::cleanup_expired_helper_scripts();
+}
+add_action( 'jetpack_backup_cleanup_helper_scripts', 'jetpack_backup_cleanup_helper_scripts' );
```

Comment:
Can we not call the method directly? Instead of wrapping in into a function?

Reply:
Yes, you are right. This is redundant. I have removed the function, and made the action directly call into Helper_Script_Manager.

... I didn't realize autoload worked with hooks, but there you go. Tested, and it works. Thanks for the suggestion! :)